astrobase/periodbase/__init__.py

```diff
@@ -199,6 +202,7 @@ def independent_freq_count(frequencies, times, conservative=True):
 from .smav import aovhm_periodfind
 from .macf import macf_period_find
 from .kbls import bls_serial_pfind, bls_parallel_pfind
+from .tls import tls_parallel_pfind
```

Comment:
Might want to make this import optional, if we don't want to make the `transitleastsquares` package a dependency. (In environments w/out it, `from astrobase import periodbase` would raise [this error msg]( https://github.com/waqasbhatti/astrobase/pull/84/files#diff-1338e87859cb7f10b207e7d65bf62938R64))